flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -291,4 +292,12 @@ class AndroidNotificationDetails {
   /// For most styles, the colouring will only be applied if the notification is
   /// for a foreground service notification.
   final bool colorized;
+
+  /// Set custom notification count.
+  ///
+  /// For Android 8.0 or newer,
```

Comment:
Could you remove this line please? Putting this here means it serves no purpose on Android versions older than 8 but that's not true if you had read the API docs I shared before and tried it on older Android versions. For convenience this is what is said

```
Sets the number of items this notification represents. On the latest platforms, this may be displayed as a badge count for Launchers that support badging. Prior to [O](https://developer.android.com/reference/android/os/Build.VERSION_CODES.html#O--) it could be shown in the header. And prior to [N](https://developer.android.com/reference/android/os/Build.VERSION_CODES.html#N--) this was shown in the notification on the right side.
```

If you run the code on emulators running an older Android version, you'll see what is described there e.g. the following is from Android 4.3

![image](https://user-images.githubusercontent.com/25263378/178130652-ac4c546c-80aa-4322-bdea-9a72a990cbb6.png)

Reply:
Thank you for pointing out. It has been removed.